include/service/json_fs_entity_service.js

```diff
@@ -74,7 +74,7 @@ module.exports = function JSONFSEntityServiceModule(pb) {
      */
     JSONFSEntityService.prototype.set = function(key, value, cb) {
         if (!util.isObject(value) && !util.isArray(value)) {
-            cb(new PBError(this.type+": Value must be an array or object: "+util.inspect(value)), null);
+            cb(new pb.PBError(this.type+": Value must be an array or object: "+util.inspect(value)), null); // PBError class necessary?
```

Comment:
I do not believe it is.  At one point, early on, there was special logic to handle these errors.  That is no longer the case.  However, without tests there is not way to no for sure.  I would just leave the comments in for now.